api/godog/requests.go

```diff
@@ -82,6 +82,26 @@ func (api *BaseFeature) SetRequestBodyParameterTo(key, value string) (err error)
 	return
 }
 
+func (api *BaseFeature) SetRequestBodyParameterToInt(key string, value int) (err error) {
+	prevMap := api.Request.Body
```

Comment:
Perhaps support getValue in this method?
```
if strings.HasPrefix(value, ".") {
		if value, err = api.GetValue(value); err != nil {
			return err
		}
	}
```
GetValue returns string, so conversion to int is most likely needed.